src/IBM.Watson.SpeechToText.v1/SpeechToTextService.cs

```diff
@@ -2647,7 +3520,7 @@ public DetailedResponse<object> AddGrammar(string customizationId, string gramma
             {
                 grammarName = Uri.EscapeDataString(grammarName);
             }
-            if (string.IsNullOrEmpty(grammarFile))
+            if (grammarFile == null)
```

Comment:
Need to address this in the generator - string.isnullorempty is currently generated but grammar file is a memorystream